### PR TITLE
Update example API

### PR DESCRIPTION
**Pull request by anthmatic**
Updating the quote api used to: https://premium.zenquotes.io

### CODE REVIEW

File: src/app/app.functions/crm-card.js [hunk 1]
```diff
@@ -1,23 +1,24 @@
 const axios = require("axios");
 
 exports.main = async (context = {}, sendResponse) => {
-  const fallbackQuote = `I think I do myself a disservice by comparing myself to Steve Jobs and Walt Disney and human beings that we've seen before. It should be more like Willy Wonka...and welcome to my chocolate factory.`;
```
Comment (anthmatic) [inline src/app/app.functions/crm-card.js:4]: I don't think we need a fallback?

File: src/app/app.functions/crm-card.js [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
+    const { data } = await axios.get("https://zenquotes.io/api/random");
 
     sendResponse({
       results: [
         demoObject,
         {
           objectId: 2,
-          title: "Wisdom from Kanye",
-          quote: data.quote || fallbackQuote,
+          link: "https://zenquotes.io/",
```
Comment (anthmatic) [inline src/app/app.functions/crm-card.js:18]: I wonder if there is a better way to include a link than this?

File: src/app/app.functions/crm-card.js (hunk 1 above)
Comment (fndicu) [reply src/app/app.functions/crm-card.js:4]: agreed, we can throw the error message instead

File: src/app/app.functions/crm-card.js (hunk 2 above)
Comment (fndicu) [reply src/app/app.functions/crm-card.js:18]: According to their docs I think that this is sufficient.  " API Usage Limits and Attribution...We require that you show attribution with a link back to https://zenquotes.io/ when using the free version of this API.

**Review by fndicu — APPROVED** [review]

**Pull request merged**